spec/change_set_persisters/plum_change_set_persister_spec.rb

```diff
@@ -432,15 +432,14 @@
 
   describe "collection interactions" do
     context "when a collection is deleted" do
-      it "cleans up associations from all its members" do
+      it "deletes members" do
         collection = FactoryGirl.create_for_repository(:collection)
         resource = FactoryGirl.create_for_repository(:scanned_resource, member_of_collection_ids: collection.id)
         change_set = CollectionChangeSet.new(collection)
 
         change_set_persister.delete(change_set: change_set)
-        reloaded = query_service.find_by(id: resource.id)
 
-        expect(reloaded.member_of_collection_ids).to eq []
+        expect { query_service.find_by(id: resource.id) }.to raise_error(Valkyrie::Persistence::ObjectNotFoundError)
```

Comment:
I don't think we want to delete objects that link to a collection when the collection is deleted.  I think the old behavior is what we should do (the collection id should be removed from the member_of_collection_ids property, but the object should remain).